lib/jxl/modular/transform/rct.cc

```diff
@@ -15,7 +22,25 @@ void InvRCTRow(const pixel_type* in0, const pixel_type* in1,
                 "Invalid transform type");
   int second = transform_type >> 1;
   int third = transform_type & 1;
-  for (size_t x = 0; x < w; x++) {
+
+  size_t x = 0;
+  if (transform_type == 6) {
+    const HWY_FULL(pixel_type) d;
+    const size_t N = Lanes(d);
+    for (; x + N - 1 < w; x += N) {
+      auto Y = Load(d, in0 + x);
+      auto Co = Load(d, in1 + x);
+      auto Cg = Load(d, in2 + x);
+      Y -= ShiftRight<1>(Cg);
+      auto G = Cg + Y;
+      Y -= ShiftRight<1>(Co);
+      auto R = Y + Co;
+      Store(R, d, out0 + x);
+      Store(G, d, out1 + x);
+      Store(Y, d, out2 + x);
+    }
+  }
```

Comment:
While we're at it, can we also SIMDfy the other case? it shouldn't be hard.

Reply:
Done.